NetKet/Machine/sum_output.hpp

```diff
@@ -84,30 +84,20 @@ class SumOutput : public AbstractLayer<T> {
 
   void SetParameters(VectorConstRefType /*pars*/, int /*start_idx*/) override {}
```

Comment:
I never liked these SetParameters and GetParameters with two arguments... Can't we just use an Eigen:Map as you have done in other parts? https://eigen.tuxfamily.org/dox/group__TutorialReshapeSlicing.html

Reply:
sure that is possible.